docs/articles/basics/first_bot.md

```diff
@@ -43,7 +43,7 @@ Locate the *solution explorer* on the right side, then right click on `Dependenc
 <br/>
 You'll then be greeted by the NuGet package manager.
 
-Select the `Browse` tab towards the top left, then type `DSharpPlus` into the search text box.
+Select the `Browse` tab towards the top left, then type `DSharpPlus` into the search text box with the Pre-release checkbox checked **ON**.
```

Comment:
I wouldn't specify that they *need* to have prerelease checked, perhaps instead say something like "Our latest up-to-date versions can be found by ticking the Pre-release checkbox".

Reply:
so when 4.x gets released I totally agree that should be the case but until we have a stable non-pre-release version, i think we should have this since we really haven't supported anyone with questions on 3.x and none of our documentation works in 3.x anymore.  However, if u still want that changed after this statement, I will make the change.

Reply:
With the amount of support that we had today in regards to ppl installing 3.2 and using 4.x documentation, was the reason for this wording

Reply:
Well we'll have to remember to change this once 4.x is fully released then. If no one else has objections to that I guess that's fine.